### PR TITLE
[TechRadar] Deprecate RadarEntry.url

### PR DESCRIPTION
**Pull request by legendik**
## Hey, I just made a Pull Request! 👋

In the pull request https://github.com/backstage/backstage/pull/15802 i've introduced `links` attribute, so you can add more links to the tech radar's entry description (documentation, external tools...). But i forget to deprecate `url` attribute.

In some next major version, we can remove `RadarEntry.url` and just use more robust `RadarEntry.links`.

#### :heavy_check_mark: Checklist

- [x] A changeset describing the change and affected packages. ([more info](https://github.com/backstage/backstage/blob/master/CONTRIBUTING.md#creating-changesets))
- [ ] Added or updated documentation
- [ ] Tests for new functionality and regression tests for bug fixes
- [ ] Screenshots attached (for UI changes)
- [x] All your commits have a `Signed-off-by` line in the message. ([more info](https://github.com/backstage/backstage/blob/master/CONTRIBUTING.md#developer-certificate-of-origin))

### CODE REVIEW

File: .changeset/hip-chairs-double.md [hunk 1]
```diff
@@ -0,0 +1,5 @@
+---
+'@backstage/plugin-tech-radar': minor
+---
+
+Deprecate `RadarEntry.url` - use `RadarEntry.links` instead
```
Comment (benjdlambert) [inline .changeset/hip-chairs-double.md:5]: Maybe an example of what the migration should look like would be good here, a before and after :)

Comment (legendik) [reply .changeset/hip-chairs-double.md:5]: Good idea, added migration diff. Also updated `sample.ts` to use the `links` attribute instead of `url`.

**Review by benjdlambert — CHANGES_REQUESTED** [review]
Sorry - just thought that we should probably do this too, so that we can drop the `url` entirely if you're not using it.

File: .changeset/hip-chairs-double.md [hunk 2]
```diff
@@ -0,0 +1,20 @@
+---
+'@backstage/plugin-tech-radar': minor
```
Comment (benjdlambert) [inline .changeset/hip-chairs-double.md:2]: Think that this should also be a `patch` change too, as the `tech-radar` plugin is `0.x.x`. So breaking changes would be `minor` in this case.

**Review by benjdlambert — APPROVED** [review]

**Pull request merged**